cmd/snap/cmd_booted.go

```diff
@@ -46,6 +47,11 @@ func (x *cmdBooted) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	if release.OnClassic {
+		fmt.Fprintf(Stdout, "Ignoring 'booted' on classic")
```

Comment:
The change looks fine as it's just moving code around, but this still feels like legacy that should go away. We have snapd itself running on boot, and we have a state that is empty / not present to tell us when we did run or not. All of this external "first boot" logic feels unnecessary by now, given that.

Still, feel free to merge it and we can fix that later.